robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -227,7 +227,7 @@ function initRosWeb () {
     } else {
       if (voltage < MIN_VOLTAGE + VOLTAGE_WARNING) {
         textColor('#battery-voltage', 'orange')
-
+        $('#battery-temperature').notify('Warning! Voltage is approaching danger value!', 'warn',{position:"bottom"})
```

Comment:
Why does `#battery-temperature` get the **Voltage** warning notification?

Reply:
#battery-temperature is the Celsius label to the right of the battery logo and below the battery voltage #battery-voltage, not to get confused with the 3 battery temperature logos on the top-right of the page. #battery-temperature gets the voltage because if #battery-voltage receives it then it hides the battery temperature #battery-temperature and other elements

Reply:
This is a bit confusing, perhaps you should create a parent div with a more appropriate name. Glancing in the code, it looks like an error. I wouldn't be surprised if someone would try to change it to #battery-voltage thinking it was an error

Reply:
Ok I added a new inner div 'battery-voltage-warning' and passed the warning notification through there. The location of the pop up is still the same

Reply:
@MartensCedric